### ISSUE TITLE
API query args to specify blink1_id not congruent

### ISSUE THREAD

**Issue by jhein05** [todbot/Blink1Control2#141]
The way you pass a blink1_id through the API is not congruent between different commands. It expects "blink1_id" in some cases and "blink1id" in others. Examples:

```
curl 'http://localhost:8934/blink1/on?blink1_id=1'
curl 'http://localhost:8934/blink1/pattern/play?pname=policecar&blink1id=1'
```

**Comment by todbot** [comment]
Hi! Thanks for catching this, I feel so embarrassed.  I'm queuing up some updates for a new release hopefully next week.

**Comment by jhein05** [comment]
Haha, now I feel embarrassed, that I embarrassed you. Oh, when you touch things anyway: At least for me, it was also not really clear how to use the id and the ledn from the API docs and examples. I saw "blink1_id" mentioned there, but I thought it was an IFTTT thing and whether to use 0 and 1 or the devices serial numbers. I think one example, like the ones I posted above, would make it easier for people.
